content/language/word-list/man-in-the-middle.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Man-in-the-middle attack
+---
+
+**Reasoning:** 
+
+When an attacker secretly intercepts and relays communications between two systems or people who believe that they are communicating directly with each other.
+
+Gendered language that can perpetuate bias and stereotypes.
+Implies that women do not have the skills to perpetrate this type of hacking.
+
+
+**Recommendation:** Adopt immediately.
```

Comment:
Tier 3 is "consider replacement" rather than Adopt immediately (came over from the 'cripple' file that was used as a template).
Thanks for doing this request and recording the session! Cathi

Reply:
> Tier 3 is "consider replacement" rather than Adopt immediately (came over from the 'cripple' file that was used as a template). Thanks for doing this request and recording the session! Cathi

Cathi is right. Eric, can you change it, please?

Reply:
> > Tier 3 is "consider replacement" rather than Adopt immediately (came over from the 'cripple' file that was used as a template). Thanks for doing this request and recording the session! Cathi
> 
> Cathi is right. Eric, can you change it, please?

Here's how you can cue up that fix for @ericbetts to approve in his branch (which updates the PR):

1. Click on the symbol on the toolbar above the comment box with the `[+-]` symbol.
2. This creates an area in the active comment box beginning with three back-ticks and `suggestion` containing the content _as it is right now_.
3. Change the content in the area below the `suggestion` and the closing three back-ticks. If you click "Review" you can see how the suggestion is rendered.
4. Once you save your comment, the suggestion can be merged into the branch directly from a button below the suggestion, so @ericbetts can see and approve in one go.

Like this:

```suggestion
**Recommendation:** Consider replacement.
```

The reasons for doing these extra steps:
- Makes it clear exactly what change you are asking for.
- Attributes the change/request back to the suggester.
- Gives a way to engage with the PR author in a back-and-forth over suggestions.
- More efficient.

Reply:
Thanks, @quaid. I'm not sure that I did this correctly because the symbol in the toolbar is a different, but it did indicate a suggestion so...hopefully @ericbetts is able to accept it. Hard to believe I used to edit entire documents in this tool.
Thanks, Cathi